app/code/Magento/Shipping/Model/Config/Source/Allmethods.php

```diff
@@ -65,4 +79,24 @@ public function toOptionArray($isActiveOnlyFlag = false)
 
         return $methods;
     }
+
+    /**
+     * Get Current Admin store to Display only available methods for store.
+     *
+     * @return int
+     */
+    protected function getAdminStoreId()
```

Comment:
Please, use `private` scope when introducing a new method

Reply:
ok. Thanks. I will update that.